aiohttp/client_ws.py

```diff
@@ -235,11 +239,32 @@ async def receive(self, timeout: Optional[float] = None) -> WSMessage:
 
             try:
                 self._waiting = self._loop.create_future()
+
+                read_task: asyncio.Task[WSMessage] = asyncio.create_task(
+                    self._reader.read()
+                )
+                is_heartbeat_failed_task: asyncio.Task[bool] = asyncio.create_task(
+                    self._is_heartbeat_failed.wait()
+                )
```

Comment:
Do we really need two additional tasks in order to solve this bug? Is it at all necessary to check heartbeat status, or is it sufficient to check transport.is_closing()?